src/triage/component/postmodeling/contrast/parameters.py

```diff
@@ -0,0 +1,29 @@
+"""
+Postmodeling parameters
+
+This script contain the parameters Class that will be used across all the
+postmodeling functions within the ModelEvaluator and ModelGroupEvaluator
+classes. This class will be initialized using the 'postmodeling_parameters.yaml' 
+file.abs
+
+"""
+
+import yaml
+import ast
+
+class PostmodelParameters(object):
+    '''
+    PostmodelParameters reads all parameters from a 'yaml' file and store them
+    in this object to be used in other functions. Different metrics can be
+    passed to this object, by default it will reads from a
+    'postmodeling_parameters.yaml', but an Audition config file can be passed
+    and will parse from it the needed parameters
+    '''
+    def __init__(self, path_params):
+
+        with open(path_params) as f:
+            params = yaml.load(f)
+        self.__dict__.update(params)
+        self.figsize = ast.literal_eval(self.figsize)
```

Comment:
This is about as unnecessary as `eval` :stuck_out_tongue: 

Are you just trying to get a pair of integers from YAML?

YAML doesn't _directly_ support tuples but why not just:

    figsize: [12, 12]

You'll have a pair defined by a list, rather than a tuple … but, that's probably fine.

If it's important that it be a tuple, I would _strongly_ encourage that here you simply do:

    self.figsize = tuple(self.figsize)

Or, you _can_ [specify Python tuples in YAML](https://stackoverflow.com/questions/39553008/how-to-read-a-python-tuple-using-pyyaml)! It's just a little bit ugly:

    figsize: !!python/tuple [12, 12]